SR2019/2019-03-11-competition-information.md

```diff
@@ -0,0 +1,53 @@
+---
+to: Student Robotics 2019
+subject: Competition Informatiom
+---
+
+Hi all,
+
+There's just 26 days between now and the competition, and we're getting the final things ready for Student Robotics 2019. We hope your robots are battle-ready, and you're well on your way to entering a robot capable of winning!
+
+We have some final information for you for the competition.
+
+## Reduced mobility
```

Comment:
Not sure this is a great title for this section. The intent is clear enough from the section, however at first it reads a little oddly.

I can't immediately think of a better phrasing, though maybe something like "Ease of access" or "Venue layout"? The information that the the venue is laid out over several floors is relevant to everyone.